Formula/glibc.rb

```diff
@@ -195,4 +208,14 @@ def post_install
     safe_system "#{lib}/libc.so.6", "--version"
     safe_system "#{bin}/locale", "--version"
   end
+
+  def ld_so_conf
+    <<~EOS
+      #{lib}
+      #{HOMEBREW_PREFIX}/opt/gcc/lib/gcc/current
```

Comment:
We may move this to `etc/ld.so.conf.d/00-homebrew-gcc.conf` rather than put it in `etc/ld.so.conf`. Note that the wildcard `*.conf`s are included in alphabetical order. Do we need also add a README in `ld.so.conf.d` directory?